external-events/pnp/pnp.public.output.change-notification-events.v1.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$id": "pnp.public.output.change-notification-events.v1", 
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "$ref" : "#/definitions/record:hiiretail.contracts.changenotifications.ChangeNotificationEvent",
+  "title": "HiiRetail Change Notification Events",
+  "description": "Publishes Notification Events for changes in entities",
+  "definitions" : {
+    "record:hiiretail.contracts.changenotifications.ChangeNotificationEvent" : {
+      "description" : "Payload describes the entity that has been changed",
+      "type" : "object",
+      "required" : [ "correlationId", "tenantId", "businessUnitIds", "entityType", "entityId", "type", "timestamp" ],
+      "additionalProperties" : false,
+      "properties" : {
+        "correlationId" : {
+          "type" : "string"
+        },
+        "tenantId" : {
+          "type" : "string"
+        },
+        "businessUnitIds" : {
+          "default" : [ ],
+          "type" : "array",
+          "items" : {
+            "type" : "string"
+          }
+        },
+        "entityType" : {
+          "type" : "string"
+        },
+        "entityId" : {
+          "type" : "string"
+        },
+        "type" : {
+          "$ref" : "#/definitions/enum:hiiretail.contracts.changenotifications.ChangeNotificationEventType"
+        },
+        "timestamp" : {
+          "type" : "integer",
```

Comment:
I think timestamp_ms is a long value

Reply:
Yes, you are right. This field is of type `long` in Avro but there is no `long` data type in [JSON Schemas](https://json-schema.org/understanding-json-schema/reference/type). But [the library](https://github.com/fge/json-schema-avro#:~:text=JSON%20Schema%20defines%20integer) that we use for generating JSON Schemas handle this by using `integer` with min and max limits which correspond to `long` value range.

Reply:
ok i understood